Editor/Data/Attribute/RestrictedAttributeDrawer.cs

```diff
@@ -0,0 +1,63 @@
+﻿namespace Zinnia.Data.Attribute
+{
+    using UnityEngine;
+    using UnityEditor;
+
+    [CustomPropertyDrawer(typeof(RestrictedAttribute))]
+    public class RestrictedAttributeDrawer : PropertyDrawer
+    {
+        public override void OnGUI(Rect position, SerializedProperty property, GUIContent label)
+        {
+            EditorGUI.BeginProperty(position, label, property);
+            RestrictedAttribute attrib = attribute as RestrictedAttribute;
```

Comment:
```suggestion
            RestrictedAttribute attrib = (RestrictedAttribute)attribute;
```
I don't like the short variable name but can't come up with something else than `restrictedAttribute` which is a bit long.